selfdrive/car/honda/values.py

```diff
@@ -93,6 +94,9 @@ class CAR:
   CAR.FIT: [{
     57: 3, 145: 8, 228: 5, 304: 8, 342: 6, 344: 8, 380: 8, 399: 7, 401: 8, 420: 8, 422: 8, 427: 3, 428: 8, 432: 7, 464: 8, 487: 4, 490: 8, 506: 8, 597: 8, 660: 8, 661: 4, 773: 7, 777: 8, 780: 8, 800: 8, 804: 8, 808: 8, 829: 5, 862: 8, 884: 7, 892: 8, 929: 8, 985: 3, 1024: 5, 1027: 5, 1029: 8, 1036: 8, 1039: 8, 1108: 8, 1322: 5, 1361: 5, 1365: 5, 1424: 5, 1600: 5, 1601: 8
   }],
+  CAR.HRV: [{
```

Comment:
This section can actually get removed.

Reply:
Yes sir, I realize that.  If it's possible to leave as a reference for other hrv'ers, as there is still some code that helps people discover their fp's.  I'm not sure the differences between HRV's at this point. 

Can we remove after further hrv users review?  Thanks

Reply:
Actually looks like we do need it for other stuff. So leaving it is fine